### PR TITLE
Add debugging option

### LINKED ISSUE #99: Add debugging option
This option should display the agent intent when updating the agenda.

### PR DESCRIPTION
**Pull request by NoB0**
Fixes #99

### CODE REVIEW

File: docs/config.md [hunk 1]
```diff
@@ -13,6 +13,7 @@ These can be either provided in a YAML config file and/or via the command line.
 * `dialogues`: Path to domain config file.
 * `intent_classifier`: Intent classifier model to be used. Only supports DialogueKit intent classifier.
 * `rasa_dialogues`: File with Rasa annotated dialogues. Only needed when using a DIET intent classifier.
+* `debug`: Boolean activating the debug mode.
```
Comment (kbalog) [inline docs/config.md:16]: Nit: "Flag (boolean) to activate debug mode."

File: docs/source/configuration.rst [hunk 2]
```diff
@@ -31,6 +31,7 @@ Simulator parameters
   * `dialogues`: Path to domain config file.
   * `intent_classifier`: Intent classifier model to be used. Only supports DialogueKit intent classifiers.
   * `rasa_dialogues`: File with Rasa annotated dialogues. Only needed when using a DIET intent classifier.
+  * `debug`: Boolean activating the debug mode.
```
Comment (kbalog) [inline docs/source/configuration.rst:34]: Same comment as above.

**Review by kbalog — APPROVED** [review]
LGTM

File: docs/source/configuration.rst (hunk 2 above)
Comment (NoB0) [reply docs/source/configuration.rst:34]: Done.

File: docs/config.md (hunk 1 above)
Comment (NoB0) [reply docs/config.md:16]: Done.

**Pull request merged**